pig/src/main/java/org/elasticsearch/hadoop/pig/EsStorage.java

```diff
@@ -118,6 +118,7 @@ public EsStorage(String... configuration) {
                 throw new EsHadoopIllegalArgumentException("Cannot parse options " + Arrays.toString(configuration), ex);
             }
         }
+        log.warn("Support for Pig has been deprecated and will be removed in a future release.");
```

Comment:
I think that including "Apache" may be a bit more clear, particularly in the context of a log.
```suggestion
        log.warn("Support for Apache Pig has been deprecated and will be removed in a future release.");
```